spec/unit/harvestdor-indexer_spec.rb

```diff
@@ -119,7 +119,7 @@
       end
     end
     context "load_whitelist" do
-      it "should not be called if there was no whitelist config setting" do
+      it "not called if there was no whitelist config setting" do
```

Comment:
thanks @mjgiarlo.  that's a great tip.  IN this case, I changed them by hand, foolishly.  :cry: